src/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -115,6 +127,8 @@ function VisEditor($scope, $route, timefilter, AppState, $location, kbnUrl, $tim
     docTitle.change(savedVis.title);
   }
 
+  // Extract visualization state with filtered aggs.
+  // Consists of: aggs, params, listeners, title, and type.
```

Comment:
Sounds good but not sure what you mean re point out where the reader can find current state. Can you give me an example of the comment you have in mind?